requirements.txt

```diff
@@ -4,4 +4,5 @@ openai==0.27.7
 azure-search-documents==11.4.0b6
 azure-storage-blob==12.17.0
 python-dotenv==1.0.0
-azure-cosmos==4.5.0
\ No newline at end of file
+azure-cosmos==4.5.0
```

Comment:
wasn't this requirement in the project already for the conversation history feature?

Reply:
It's probably triggering on the new whitespace/newline added at the end.